network.go

```diff
@@ -1003,3 +1043,22 @@ func (n *network) deriveAddressSpace() (string, error) {
 	}
 	return ipd.defaultLocalAddressSpace, nil
 }
+
+func (n *network) Info() NetworkInfo {
+	return n
+}
+
+func (n *network) Labels() []string {
+	n.Lock()
+	defer n.Unlock()
+	if n.generic != nil {
+		if m, ok := n.generic[netlabel.GenericData]; ok {
+			return netlabel.FromMap(m.(map[string]string))
+		}
+	}
+	return []string{}
+}
+
+func (n *network) Scope() string {
+	return n.scope
```

Comment:
After all the recent changes, is the scope populated ?
maybe we should remove this variable ?

Can you please add a UT for this ?